spec/index.adoc

```diff
@@ -990,7 +1000,7 @@ The comment after each function lists from where these functions may be invoked:
 * `C`: from a cleanup callback
 * `s`: the `(start)` module initialization function
 * `F`: from `canister_inspect_message`
-* `H`: from `canister_heartbeat`
+* `H`: from `canister_heartbeat` or `canister_global_timer`
```

Comment:
Inside the code we've decided to rename this `heartbeat` or `global timer` into `system task`, as they share a lot of code. Not sure if it's relevant for the spec, but JFYI.

Reply:
I see 4 occurrences of the regular expression `heartbeat.*or.*timer` scattered throughout the spec. We could abbreviate, but then we'd need to introduce the abbreviation properly so I'm not sure about this change.